import/popups/ShareTextPage.qml

```diff
@@ -13,40 +13,20 @@ import QtQuick 2.0
 import Sailfish.Silica 1.0
 import Sailfish.TransferEngine 1.0
 
-Page {
+SharePage {
     id: root
 
     property string text
     property string type: "text/plain"
-    property string icon: "icon-launcher-browser"
 
-    ShareMethodList {
-        id: shareMethodList
-
-        anchors.fill: parent
-        header: PageHeader {
-            //: List header for sharing method list (generic)
-            //% "Share"
-            title: qsTrId("sailfish_components_webview_popups-he-share")
-        }
-        content: {
-            "name": text,
-            "data": text,
-            "type": type,
-            "icon": icon,
```

Comment:
Don't recall but is this one of those deprecated fields?

Reply:
Haven't spotted any usage and no one else seems to have idea why this has been there :)